tests/test_cases/configure-conjur-identity/playbook.yml

```diff
@@ -2,7 +2,7 @@
 - name: Configuring conjur identity on remote hosts
   hosts: testapp
   roles:
-    - role: configure-conjur-identity
+    - role: cyberark.conjur
```

Comment:
Looking at this role and thinking about roles that might help configure a cluster, what do you think about changing this role to something more descriptive of what's happening?  Maybe `cyberark.conjurize`, or `cyberark.conjur_identity